sql-db/sql-app-compatibility/src/test/java/io/quarkus/ts/sqldb/compatibility/DB2DatabaseIT.java

```diff
@@ -12,6 +12,7 @@
 @QuarkusScenario
 @DisabledOnNative(reason = "Compatibility mode check in JVM mode is enough for this DB")
 @Tag("fips-incompatible") // Reported in https://github.com/IBM/Db2/issues/43
+@Tag("podman-incompatible") //TODO https://github.com/containers/podman/issues/16432
```

Comment:
The issue has been closed as not a bug .
We should make sure Quarkus engineering has this still on their radar, maybe comment in https://github.com/quarkusio/quarkus/issues/31253. WDYT?

Reply:
It is a bug, related to DB2 image (and not podman _per se_). The image we use is not supported anymore, so I suggest to switch to the official one[1] sometime later. Given uncertain future of DB2 and Quarkus, maybe we should just ignore this. And this is mentioned in the comment[2] by the way :)

[1] https://www.ibm.com/docs/en/db2/11.5?topic=system-linux
[2] https://github.com/quarkusio/quarkus/issues/31253#issuecomment-1449564257

Reply:
Ok, I saw the comment, it just felt like long time since any activity on that issue.

Switching to the official image is a good idea, although it wouldn't help with this issue, as the official one seems to behave the same. But I did just a quick test.

I agree with leaving this as is for now.